rules/explain-deleted-or-modified-appointments/rule.md

```diff
@@ -22,8 +22,15 @@ guid: 1c278726-9df4-44dc-b09b-bb17470c31ad
 
 ---
 
+Every change to an appointment is a hassle and it's good to be explicit as to why the appointment had to change. There are also people that need some pressure put on them when they change an appointment multiple times so that we don't end up on V7 🤣 (see Version Numbers below).
```

Comment:
Hey @SamBWagner 

1. Can you re-word the sentence below? I didn't quite get the message

> There are also people that need some pressure put on them when they change an appointment multiple times so that we don't end up on V7 🤣 (see Version Numbers below).